nova/controller.sls

```diff
@@ -90,8 +90,9 @@ nova_placement_package:
   pkg.installed:
   - name: nova-placement-api
 
-/etc/apache2/sites-available/nova-placement-api.conf:
+nova_placement_apache_configs:
   file.managed:
+  - names: [/etc/apache2/sites-available/nova-placement-api.conf, /etc/apache2/sites-enabled/nova-placement-api.conf]
```

Comment:
In sites-enabled are supposed to be symlinks to sites-available. This setup is Debian-specific and will not work on non-Debian families (eg. RHEL). Also there's `a2ensite` tool that can do this and AFAIK similar salt resource to manage this Debian-specific.